src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -51,7 +62,19 @@
   private static final String USER_FRIENDS_LIST_PROPERTY = "friends-list";
   private static final String USER_ID_PROPERTY = "id";
   private static final String USER_NAME_PROPERTY = "name";
+  private static final String USER_PROFILE_PHOTO = "profile-photo";
+  private static final String USER_PHOTO_2 = "photo-2";
+  private static final String USER_PHOTO_3 = "photo-3";
+  private static final String USER_PHOTO_4 = "photo-4";
+  private static final String USER_PHOTO_5 = "photo-5";
```

Comment:
Can we store these as a `List<>` instead of N hard-coded fields?

We can still enforce a limit on the list size, but it would reduce duplication and make it easier to change the number of images in the future.

Reply:
Changed it to a list!